include/tscpp/util/LocalBuffer.h

```diff
@@ -0,0 +1,49 @@
+/** @file
+
+   LocalBuffer
+
+   @section license License
+
+   Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+
+#pragma once
+
+namespace ts
+{
+template <std::size_t EstSizeBound = 1024> class LocalBuffer
+{
+public:
+  LocalBuffer(std::size_t size) : _ptr(size == 0 ? nullptr : (size > EstSizeBound ? new uint8_t[size] : _buf)) {}
+  ~LocalBuffer()
+  {
+    if (_ptr && _ptr != _buf) {
+      delete[] _ptr;
+    }
+  }
+
+  uint8_t *
+  get() const
```

Comment:
Make this `data` to be consistent with STL nomenclature. Having a `size` method would be good.

Reply:
done